src/Microsoft.VisualStudio.Threading.Tests/TestBase.cs

```diff
@@ -198,7 +198,7 @@ protected async Task CheckGCPressureAsync(Func<Task> scenario, int maxBytesAlloc
                         if (SingleThreadedTestSynchronizationContext.IsSingleThreadedSyncContext(SynchronizationContext.Current))
                         {
                             var frame = SingleThreadedTestSynchronizationContext.NewFrame();
-                            SynchronizationContext.Current.Post(state => frame.Continue = false, null);
+                            SynchronizationContext.Current!.Post(state => frame.Continue = false, null);
```

Comment:
📝 This is not necessary once the `IsSingleThreadedSyncContext` method is annotated as a null check

```suggestion
                            SynchronizationContext.Current.Post(state => frame.Continue = false, null);
```